osama-breakout-room/ArvinAlhissou/style.css

```diff
@@ -0,0 +1,137 @@
+* {
+    padding: 0px;
+    margin: 0px;
+    box-sizing: border-box;
+    background-color: rgb(235, 229, 229);
+  }
+  
+  header {
+    width: 100%;
+    height: 60px;
+    box-shadow: 0px 0px 9px 0px grey;
+    padding: 6px;
+    background-color:rgb(235, 229, 229);
+  }
+  .wrapper {
+    width: 800px;
+    margin: 0 auto;
+    background-color:white ;
+    
+  }
+  
+
+  .logo {
+    float: left;
+  }
+  .logo img {
+    width: 50px;
+  }
+  nav {
+    float: right;
+  }
+  .btn {
+    
+    display: inline-block;
+    border-radius: 12px;
+    padding: 10px 18px 10px 18px;
+    margin: 6px;
+    text-align: center;
+    font-size: 11px;
+    font-family: Arial, Helvetica, sans-serif;
+    text-decoration: none;
+  }
+  .btn-bordered:hover{
+    padding: 11px 19px 11px 19px;
+    box-shadow: 0px 0px 5px 0px #1da1f2;
+  }
+  .btn-bordered {
+    border: 1px solid #1da1f2;
+    color: #1da1f2;
+  }
+  .btn-blue {
+    background-color: #1da1f2;
+    border: 1px solid #1da1f2;
+    color: white;
+  }
```

Comment:
Great job on your approach for styling the buttons. 👌
By your approach, you are maintaining the reusability of your code as you have a class called "btn" for giving all your buttons the same shape while having classes for giving buttons specific colors and looks.